server/repository/public/user/update.ts

```diff
@@ -28,6 +28,19 @@ export const update = async (
     [institution, lang, name, status, position, email, id]
   )
 
+  await client.query(`delete from users_role WHERE user_id = $1`, [id])
+
+  const userRolePromises = roles
+    .filter((userRole) => !!userRole)
+    .map((userRole: UserRole<RoleName>) =>
+      client.query(
+        `insert into users_role (user_id, assessment_id, cycle_uuid, country_iso, role, accepted_at) values ($1, $2, $3, $4, $5, now())`,
+        [id, userRole.assessmentId, userRole.cycleUuid, userRole.countryIso, userRole.role]
+      )
+    )
+
+  await Promise.all(userRolePromises)
```

Comment:
we need to ensure only admin can updates the roles. add this check in the controller.

Plus, we need to handle the 'deactivate' functionality.